ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -394,6 +396,18 @@ function WorkloadCvesOverviewPage() {
         />
     );
 
+    // Report-specific state management
+    const [isCreateReportDropdownOpen, setIsCreateReportDropdownOpen] = useState(false);
+    const [isCreateOnDemandReportModalOpen, setIsCreateOnDemandReportModalOpen] = useState(false);
+    const isOnDemandReportsEnabled = isFeatureFlagEnabled('ROX_VULNERABILITY_AD_HOC_REPORTS');
```

Comment:
Disregard the old "ad-hoc" verbiage. I will change the feature flag env variable in the next PR. It will be called "ROX_VULNERABILITY_ON_DEMAND_REPORTS"